### PR TITLE
ROOST.AI Integration

### PR DESCRIPTION
**Pull request by Divyeshzb**
This PR introduces a new integration with [Roost](https://roost.ai), Ephemeral DevOps Platform.

**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [x] I have updated my user documentation

**Availability**
Roost.ai is partner to LaunchDarkly. 

Roost provide a third party integration of LaunchDarkly in UI in its EaaS view where user can manage LaunchDarkly Project , Environment and feature flag directly from Roost UI.

**Request preview**

`➜  integration-framework git:(main) ✗ npm run preview roost.ai

> @launchdarkly/integration-framework@1.0.0 preview /Users/divyesh/go/src/github.com/roost-io/integration-framework
> node preview.js "roost.ai"

The "roost.ai" integration does not make any outbound requests from LaunchDarkly.`

We already created a [confluence documentation](https://zbio.atlassian.net/wiki/spaces/RFD/pages/1748500481/Roost+LaunchDarkly+Integration) for the user to enable Launch Darkly Integration in Roost.

Let me know if any question.Thankyou

### CODE REVIEW

**Comment by Divyeshzb** [comment]
Thankyou @InTheCloudDan for the changes request. Made the changes for same.

File: integrations/roost.ai/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "author": "Roost AI",
+  "supportEmail": "support@zb.io",
+  "links": {
+    "site": "https://roost.ai/",
+    "privacyPolicy": "https://roost.ai/privacy-policy",
+    "supportWebsite": "https://zbio.atlassian.net/wiki/spaces/RFD/pages/1748500481/Roost+LaunchDarkly+Integration"
+  },
+  "categories": ["developer-tools", "automation"],
+  "icons": {
+    "square": "assets/images/square.svg",
```
Comment (InTheCloudDan) [inline integrations/roost.ai/manifest.json:15]:
```suggestion
    "square": "assets/square.svg",
```

File: integrations/roost.ai/manifest.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "horizontal": "assets/images/horizontal.svg"
```
Comment (InTheCloudDan) [inline integrations/roost.ai/manifest.json:16]:
```suggestion
    "horizontal": "assets/horizontal.svg"
```

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]
@Divyeshzb Can you accept these additional suggestions and I'll be able to get this merged in today, thanks!

**Comment by Divyeshzb** [comment]
@InTheCloudDan Done the suggested changes. Thanks!!

**Review by InTheCloudDan — APPROVED** [review]

**Comment by InTheCloudDan** [comment]
@Divyeshzb I thought the images were the only changes but EOF is failing. Can you add a newline at end of the manifest file? And can you also add in the manifest: `"otherCapabilities": ["external"]`

**Comment by Divyeshzb** [comment]
@InTheCloudDan added the otherCapabilities key and also ran prettier:check script for the manifest.

**Comment by Divyeshzb** [comment]
@InTheCloudDan I have resolved EOF error in roost.ai integration images. Now according to logs cloudquery integration have the issues.

**Pull request closed without merging**